python/datasources/cdc_hiv.py

```diff
@@ -0,0 +1,228 @@
+        f'cdc_hiv/{NATIONAL_LEVEL}', f'{breakdown}_{NATIONAL_LEVEL}_{SOURCE_YEAR}.csv', skiprows=9, thousands=",")
+    nat_from_state_df['FIPS'] = '00'
+    nat_from_state_df['Geography'] = 'United States'
+
+    return nat_from_state_df
+
+
+class CDCHIVData(DataSource):
+
+    @ staticmethod
+    def get_id():
+        return 'CDC_HIV_DATA'
+
+    @ staticmethod
+    def get_table_name():
+        return 'cdc_hiv_data'
+
+    def upload_to_gcs(self, gcs_bucket, **attrs):
+        raise NotImplementedError(
+            'upload_to_gcs should not be called for CDCHIVData'
+        )
+
+    def write_to_bq(self, dataset, gcs_bucket, **attrs):
+        for geo_level in [COUNTY_LEVEL, STATE_LEVEL, NATIONAL_LEVEL]:
+            for breakdown in [std_col.AGE_COL, std_col.RACE_OR_HISPANIC_COL, std_col.SEX_COL]:
+                table_name = f'{breakdown}_{geo_level}'
+
+                df = self.generate_breakdown_df(breakdown, geo_level)
+
+                float_cols = [std_col.HIV_POPULATION_PCT,
+                              std_col.HIV_CASES,
+                              std_col.HIV_PER_100K,
+                              std_col.HIV_PCT_SHARE
+                              ]
+
+                column_types = gcs_to_bq_util.get_bq_column_types(
+                    df, float_cols=float_cols)
+
+                gcs_to_bq_util.add_df_to_bq(
+                    df, dataset, table_name, column_types=column_types)
+
+    def generate_breakdown_df(self, breakdown, geo_level):
+        COL_NAME = std_col.COUNTY_NAME_COL if geo_level == COUNTY_LEVEL else std_col.STATE_NAME_COL
+        FIPS = std_col.COUNTY_FIPS_COL if geo_level == COUNTY_LEVEL else std_col.STATE_FIPS_COL
+
+        cols_std = {
+            'Geography': COL_NAME,
+            'FIPS': FIPS,
+            'Age Group': std_col.AGE_COL,
+            'Sex': std_col.SEX_COL,
+            'Race/Ethnicity': std_col.RACE_OR_HISPANIC_COL,
+            'Year': std_col.TIME_PERIOD_COL,
+            'Cases': std_col.HIV_CASES,
+            'Rate per 100000': std_col.HIV_PER_100K,
+            'Population': std_col.POPULATION_COL,
+        }
+
+        cols = [
+            COL_NAME,
+            FIPS,
+            std_col.TIME_PERIOD_COL,
+            breakdown,
+            std_col.HIV_CASES,
+            std_col.HIV_PER_100K,
+            std_col.HIV_PCT_SHARE,
+            std_col.HIV_POPULATION_PCT]
+
+        source_dfs = []
+        missing_data = ['Data suppressed', 'Data not available']
+
+        # county fips needs 5 digits, state digits use 2
+        format_num = 5 if geo_level == COUNTY_LEVEL else 2
+        geo = COUNTY_LEVEL if geo_level == COUNTY_LEVEL else STATE_LEVEL
+
+        for group in GROUP_DICT[breakdown].keys():
+            source_group_df = gcs_to_bq_util.load_csv_as_df_from_data_dir(
+                f'cdc_hiv/{geo}', f'{breakdown}_{demo_dict.get(group, group)}_{geo}_{SOURCE_YEAR}.csv',
+                skiprows=9, thousands=',', dtype={'FIPS': str})
```

Comment:
Also, would you mind putting comments explaining the `thousands` and `skiprows` params